integration/simulation/simulation_with_network_test.go

```diff
@@ -0,0 +1,10 @@
+package simulation
+
+import (
+	"testing"
+)
+
+// Todo - Joel
```

Comment:
You want me to write this?

Reply:
this would be the "driver" test.
Where everything is started standalone and communicates on the "official" channels